### PR TITLE
MONGOCRYPT-252 package for Debian

### PR DESCRIPTION
**Pull request by rcsanchez97**
Full Evergreen patch build (including new Linux Distro Packaging variant with debian-package-build task): https://spruce.mongodb.com/version/5f52482657e85a7177aab7de/tasks

Note that my plan is that once this merged, I will push a new branch with the same changes.  The new branch will be called `debian/1.0.4` and will start from the `1.0.4` tag.

### CODE REVIEW

File: .evergreen/config.yml [hunk 1]
```diff
@@ -115,6 +115,11 @@ functions:
           pkg_version=$mongocrypt_version
           tar -zcv -C install --transform="s|^libmongocrypt/|libmongocrypt-$pkg_version/|" --exclude=nocrypto --exclude=sharedbson -f libmongocrypt-$pkg_version.tar.gz libmongocrypt
           cd libmongocrypt/etc/
+          # The files from libmongocrypt/debian/ are the official maintainer scripts,
+          # but libmongocrypt/etc/debian/ contains a a few custom scripts that are
```
Comment (kevinAlbs) [inline .evergreen/config.yml:119]:
```suggestion
          # but libmongocrypt/etc/debian/ contains a few custom scripts that are
```

File: .evergreen/debian_package_build.sh [hunk 2]
```diff
@@ -0,0 +1,61 @@
+#!/bin/sh
+
+#
+# Test mongocxx's Debian packaging scripts.
```
Comment (kevinAlbs) [inline .evergreen/debian_package_build.sh:4]:
```suggestion
# Test libmongocrypt's Debian packaging scripts.
```

File: debian/control [hunk 3]
```diff
@@ -0,0 +1,37 @@
+Source: libmongocrypt
+Priority: optional
+Maintainer: Mongo C Driver Team <mongo-c-driver-debian@googlegroups.com>
+Uploaders: Kevin Albertson <kevin.albertson@mongodb.com>,
+           Samantha Ritter <samantha.ritter@mongodb.com>,
+           Roberto C. Sanchez <roberto@connexer.com>
```
Comment (kevinAlbs) [inline debian/control:6]: Can you also add Clyde?

File: etc/debian/control [hunk 4]
```diff
@@ -1,8 +1,10 @@
 Source: libmongocrypt
 Priority: optional
-Maintainer: Roberto C. Sanchez <roberto@connexer.com>
-Build-Depends:
-Standards-Version: 4.1.3
+Maintainer: Mongo C Driver Team <mongo-c-driver-debian@googlegroups.com>
+Uploaders: Kevin Albertson <kevin.albertson@mongodb.com>,
+           Samantha Ritter <samantha.ritter@mongodb.com>,
+           Roberto C. Sanchez <roberto@connexer.com>
```
Comment (kevinAlbs) [inline etc/debian/control:6]: Here too, add Clyde so we have all of the C/C++ team listed.

**Review by kevinAlbs — COMMENTED** [review]
Looks good from what I can tell, just a couple minor things.

File: .evergreen/config.yml (hunk 1 above)
Comment (rcsanchez97) [reply .evergreen/config.yml:119]: Done

File: debian/control (hunk 3 above)
Comment (rcsanchez97) [reply debian/control:6]: Done

File: etc/debian/control (hunk 4 above)
Comment (rcsanchez97) [reply etc/debian/control:6]: Done

**Pull request merged**